native/src/components/MapView.tsx

```diff
@@ -63,6 +62,7 @@ type MapViewProps = {
   setSheetSnapPointIndex: (index: number) => void
   followUserLocation: boolean
   setFollowUserLocation: (value: boolean) => void
```

Comment:
Do we actually still need those props? I think we only still use it to decide which location icon to display. If that is the case, we could also compare the map coordinates with the user location coordinates and decide based on that which icon to display. At least we should be able to just move it to MapView and avoid handling it in Pois.

Reply:
Not sure if tracking and comparing the map/camera coordinates is more performant and easier to handle @steffen.kleinle